rest_framework_simplejwt/serializers.py

```diff
@@ -105,25 +106,53 @@ class TokenRefreshSerializer(serializers.Serializer):
     token_class = RefreshToken
 
     default_error_messages = {
-        "no_active_account": _("No active account found for the given token.")
+        "no_active_account": _("No active account found for the given token."),
+        "password_changed": _("The user's password has been changed."),
```

Comment:
re-run the locale generation since this is a new gettext (i.e. the line where the gettext is written has moved)